src/foam/core/Window.js

```diff
@@ -225,6 +225,21 @@ foam.CLASS({
 
     function installCSS(text,  /* optional */ owner, /* optional */ id) {
       /* Create a new <style> tag containing the given CSS code. */
+      if ( text instanceof Promise ) {
+        var id = id ?? 'style' + (new Object()).$UID;
```

Comment:
`(new Object()).$UID` returns undefined?

Reply:
Returns a number for me

Reply:
You could just call foam.next$UID()